api/tests/integration/ArticleTest.php

```diff
@@ -522,9 +528,6 @@ public function shouldGetCommentsForArticleUsingWithNestedHeader()
         $client->api('discussions')->remove($discussion['Discussion']['DiscussionID']);
     }
 
-    /**
-     * @test
-     */
```

Comment:
I'm guessing disabling this test wasn't intentional, so I'll re-enable it. Which should also bring code coverage back up.